init.lua

```diff
@@ -108,12 +190,18 @@ for _, v in pairs({
 		on_construct = on_construct,
 	})
 end
+
+--- I have absolutly no idea what is the purpose but I aplied it to other stuff and it work.
 for i = 1, 5 do
-	minetest.register_alias_force("default:dry_grass_" .. i, "default:grass_" .. i)
+	minetest.register_alias_force("default:dry_grass_" .. i, "default:grass_" .. i , "default:desert_" .. i , "ethereal:dry_" .. i, "ethereal:prairie_" .. i )
```

Comment:
This is definitely wrong. register_alias_force makes all dry_grass into grass so that they can be colored using the same algorithm. This is needed for when Biome A has a "dry" variant for Biome B's nodes.

There can be only 2 parameters.

So if you want to do this for ethereal dry grass, then you need 1 extra line
if you want to do this for the 2 sand types, you need another extra call

It doesn't break anything because the API throws out parameter 3, 4 etc.